typings/custom-ckeditor-defs.d.ts

```diff
@@ -2,6 +2,15 @@
 // https://github.com/ckeditor/ckeditor-dev/issues/2947 is resolved.
 declare namespace CKEDITOR {
     interface pluginDefinition {
+        hidpi?: boolean;
+        lang?: string | string[];
         icons?: string;
```

Comment:
@NoumanMasood I'm not sure but the expected behavior for the issue #6431 was to remove these custom typedefs, right?

Reply:
So it means I have to remove these typing and replace it with 'any' type?

Reply:
@ankita240796 can you guide me please

Reply:
Hi @NoumanMasood, first of all congrats on your first PR to Oppia!

Regarding your query here, I agree with what @DubeySandeep said. To be clear, it does not mean replacing it with any types, it means removing this typing file completely (we should not need any extra type definitions for CKEditor to work correctly instead the type definitions provided for CKEditor should be sufficient).

Please feel free to ask more questions if you have any other doubt.

Thanks!

Reply:
So I have to remove the file because we do not need it anymore. right?

Reply:
@ankita240796 TO solve this issue, I have to remove the file of custom.ckeditor.def.ts ?